www/js/lib/uiUtil.js

```diff
@@ -415,9 +415,12 @@ define(rqDef, function() {
      * For each contentTheme, a stylesheet must be provided in www/css that is named 'kiwixJS' + contentTheme
      * A rule may additionally be needed in app.css for full implementation of contentTheme
      * 
-     * @param {String} theme The theme to apply (light|dark[_invert|_mwInvert])
+     * @param {String} theme The theme to apply (auto[_invert|_mwInvert]|light|dark[_invert|_mwInvert])
```

Comment:
When the user selects `Auto (wikimedia)*`, it should always display the warning `* Implements workarounds specific to Wikimedia ZIMs...` like it does when `Dark + invert (Wikimedia)*` is selected.
It does work when the OS theme is dark, but it's not displayed if the OS theme is not dark.

Reply:
I agree. I had thought this was working, but I realize now that it's because I had only checked for it when my OS was set to dark mode.